javascript/javascript1/week3/jsweek3hw.js

```diff
@@ -29,3 +29,35 @@ removeNameFromArray(nameToRemove);
 // Code done
 
 console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//Ex 2
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function getTravelTime(travelInformation) {
+  if (
+    arguments.length < 1 ||
+    !travelInformation.hasOwnProperty("destinationDistance") ||
+    !travelInformation.hasOwnProperty("speed")
+  ) {
+    return "Invalid travel information";
+  }
+  if (
+    typeof travelInformation.destinationDistance != "number" ||
+    typeof travelInformation.speed != "number"
+  ) {
+    return "Destination distance and speed must be numbers";
+  }
+  const time = travelInformation.destinationDistance / travelInformation.speed;
+  const hours = Math.floor(time);
+  const minutes = Math.floor((time % 1) * 60);
+  return `${hours} hours and ${minutes} minutes`;
+}
+console.log(getTravelTime());
+console.log(getTravelTime(2));
+console.log(getTravelTime({ destinationDistance: 100, speed: "25 " }));
+
+const travelTime = getTravelTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
```

Comment:
Great work, Reenu! 🔥

When you use two if statements in the function, they can both be executed - like they are when you run lin 59. 
You might consider, whether you want to do an if/else if statement that first takes into consideration whether the TravelInformation contains the correct properties. If it is, in fact the correct properties, you can move on to the else if statement, to check whether the properties have the right type - and provide only the error message that specifies what the user should update in the input. Just something to consider 😊